ochap-authentication/src/main/java/com/example/ochapauthentication/entities/User.java

```diff
@@ -0,0 +1,28 @@
+package com.example.ochapauthentication.entities;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "user_table")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(nullable = false, unique = true)
+    private String username;
+
+    @Column(nullable = false)
+    private byte[] password;
```

Comment:
This is not storing the password, but rather the hash of the password.

```suggestion
    private byte[] passwordHash;
```